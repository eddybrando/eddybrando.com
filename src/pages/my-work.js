import React from 'react';
import {graphql, StaticQuery} from 'gatsby';
import CompanyBlock from '../components/company-block';
import Layout from '../components/layout';
import SEO from '../components/seo';

export const PureMyWorkPage = ({data}) => {
  const companies = [
    {
      date: '2019 - Present',
      details:
        'Tech stack: JavaScript, TypeScript; Angular, Vue.js; RxJs, Vuex; Node.js; GraphQL, MongoDB',
      headline: 'IBM',
      lead: 'Senior web developer (Team lead)',
      logoImg: 'logoIbm',
    },
    {
      date: '2018 - 2019',
      details: 'Tech stack: JavaScript, JSP; Vue.js; Vuex; Java, PHP',
      headline: 'KICKZ (Zalando)',
      lead: 'Lead web developer',
      logoImg: 'logoKickz',
    },
    {
      date: '2016 - 2018',
      details: 'Tech stack: JavaScript; React, Vue.js; Redux; Node.js',
      headline: 'IBM',
      lead: 'Web developer',
      logoImg: 'logoIbm',
    },
    {
      date: '2014 - 2015',
      details: 'Tech stack: JavaScript; PHP; WordPress',
      headline: 'BodyChange',
      lead: 'Web developer',
      logoImg: 'logoBodychange',
    },
    {
      date: '2008 - 2015',
      details: 'Tech stack: JavaScript; PHP; Joomla',
      headline: 'Peruvian Embassy in Ecuador',
      lead: 'Web developer',
      logoImg: 'logoPeru',
    },
  ];

  const personalProjects = [
    {
      headline: 'Eddybrando.com',
      lead: 'Professional personal website',
      details:
        'React-based PWA generated with GatsbyJS, powered by GraphQL and tested with Jest',
      linkLabel: 'eddybrando.com',
      linkTo: 'https://eddybrando.com',
    },
    {
      headline: 'Podcasts en Español',
      lead: 'Social network for podcast enthusiasts',
      details:
        'Laravel-powered website with a PHP back end, a front end enhanced with Vue.js, and MySQL databases',
      linkLabel: 'podcastsenespanol.com',
      linkTo: 'https://podcastsenespanol.com/',
    },
    {
      headline: 'VioletaCast',
      lead: 'Podcast producer website',
      details:
        'Multilingual Laravel website written in PHP and Blade templates',
      linkLabel: 'violetacast.com',
      linkTo: 'https://violetacast.com/en',
    },
    {
      headline: 'Psicóloga Edith',
      lead: 'Professional personal website',
      details:
        'Vue.js-based website generated with Gridsome and connected to PHP Lumen micro-services',
      linkLabel: 'psicologaedith.com',
      linkTo: 'https://psicologaedith.com/',
    },
    {
      headline: 'Papel Higiénico Justo',
      lead: 'Toilet paper usage calculator',
      details: 'Calculator website built with Svelte',
      linkLabel: 'papelhigienicojusto.com',
      linkTo: 'https://papelhigienicojusto.com/',
    },
  ];

  return (
    <Layout
      headerTitle={{prepend: 'My ', highlight: 'work'}}
      mainContainerClass="main-container--full-height"
      nav={[
        {label: 'Home', link: '/'},
        {label: 'Contact me', link: '/contact-me'},
      ]}
    >
      <SEO
        description="This is my work as a developer, designer and entrepreneur specialised in web
          technologies."
        title="My work"
      />

      <section className="how-it-began">
        <h2>How it began</h2>
        <p>
          I started coding around 2003 at age 13 and drafted my first web
          designs when I was 14.
        </p>

        <p>
          Back then, my tech stack consisted of HTML 4, CSS 2 and Macromedia
          Flash 7, along with the occasional JavaScript snippet.
        </p>

        <p>
          My tools of choice were Macromedia Dreamweaver and Microsoft FrontPage
          for code editing, and Yahoo! GeoCities or the free .tk domains host
          for hosting.
        </p>
      </section>

      <section className="work-companies">
        <h2>Some companies I've worked at</h2>
      </section>

      {companies.map(({headline, date, logoImg, lead, details}, index) => (
        <CompanyBlock
          data={data}
          date={date}
          details={details}
          headline={headline}
          key={index}
          lead={lead}
          logoImg={logoImg}
        />
      ))}

      <section className="work-companies">
        <h2>A few of my personal projects</h2>
      </section>

      {personalProjects.map(
        ({headline, lead, linkLabel, linkTo, details}, index) => (
          <CompanyBlock
            details={details}
            headline={headline}
            key={index}
            lead={lead}
            linkLabel={linkLabel}
            linkTo={linkTo}
          />
        ),
      )}
    </Layout>
  );
};

const MyWorkPage = () => (
  <StaticQuery
    query={graphql`
      query {
        logoIbm: file(relativePath: {eq: "logo-ibm.jpg"}) {
          childImageSharp {
            fixed(width: 50, height: 50) {
              ...GatsbyImageSharpFixed
            }
          }
        }
        logoKickz: file(relativePath: {eq: "logo-kickz.jpg"}) {
          childImageSharp {
            fixed(width: 50, height: 50) {
              ...GatsbyImageSharpFixed
            }
          }
        }
        logoBodychange: file(relativePath: {eq: "logo-bodychange.jpg"}) {
          childImageSharp {
            fixed(width: 50, height: 50) {
              ...GatsbyImageSharpFixed
            }
          }
        }
        logoPeru: file(relativePath: {eq: "logo-peru.jpg"}) {
          childImageSharp {
            fixed(width: 50, height: 50) {
              ...GatsbyImageSharpFixed
            }
          }
        }
      }
    `}
    render={(data) => <PureMyWorkPage data={data} />}
  />
);

export default MyWorkPage;
