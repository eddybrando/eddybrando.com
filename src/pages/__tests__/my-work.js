import React from "react"
import ShallowRenderer from "react-test-renderer/shallow"

import { PureMyWorkPage } from "../my-work"

describe("MyWorkPage", () => {
  it("renders correctly", () => {
    const data = {
      logoIbm: {
        childImageSharp: {
          fixed: {
            base64:
              "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAGQABAAIDAAAAAAAAAAAAAAAAAAEDAgQF/8QAFAEBAAAAAAAAAAAAAAAAAAAAAP/aAAwDAQACEAMQAAAB4K+80EjEAH//xAAcEAACAQUBAAAAAAAAAAAAAAABAhAAAxESEzH/2gAIAQEAAQUCAJrVotuFHdcn2f/EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQMBAT8BH//EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQIBAT8BH//EABsQAAEEAwAAAAAAAAAAAAAAAAEAESExECBB/9oACAEBAAY/AoBVHEvbwu6//8QAHRAAAgICAwEAAAAAAAAAAAAAAREAMSFBIHGBkf/aAAgBAQABPyG2HQn0FUyMGKxV+EQwBZF6hMyHe+H/2gAMAwEAAgADAAAAEMAAPP/EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQMBAT8QH//EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQIBAT8QH//EAB0QAQACAgIDAAAAAAAAAAAAAAEAESExQXEQUYH/2gAIAQEAAT8QDUEatHMQaTeRb9dxsbBGkeIoHcIFV9c3MTRagVQqzO9wkiFI7Pcd+Cf/2Q==",
            width: 50,
            height: 50,
            src: "/static/b6b2d60fa4a246475c6e494ea0cb0867/9b664/logo-ibm.jpg",
            srcSet:
              "/static/b6b2d60fa4a246475c6e494ea0cb0867/9b664/logo-ibm.jpg 1x,\n/static/b6b2d60fa4a246475c6e494ea0cb0867/06a10/logo-ibm.jpg 1.5x,\n/static/b6b2d60fa4a246475c6e494ea0cb0867/f1b5a/logo-ibm.jpg 2x",
          },
        },
      },
      logoKickz: {
        childImageSharp: {
          fixed: {
            base64:
              "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAGAABAQADAAAAAAAAAAAAAAAAAAMCBAX/xAAUAQEAAAAAAAAAAAAAAAAAAAAA/9oADAMBAAIQAxAAAAHkVtU0WYkAD//EABsQAAIDAAMAAAAAAAAAAAAAAAECAAMSEBET/9oACAEBAAEFAphoQVNfWfStDY235//EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQMBAT8BH//EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQIBAT8BH//EAB0QAAICAQUAAAAAAAAAAAAAAAABAhExICEyYZH/2gAIAQEABj8CMejTyjnTNrk+nSHKq0f/xAAdEAEAAgICAwAAAAAAAAAAAAABACERQSAxUYGh/9oACAEBAAE/IYPdTzSGXhMJFXhuGvsQUUV7A9TbDo4f/9oADAMBAAIAAwAAABCzDzz/xAAUEQEAAAAAAAAAAAAAAAAAAAAg/9oACAEDAQE/EB//xAAUEQEAAAAAAAAAAAAAAAAAAAAg/9oACAECAQE/EB//xAAdEAEAAgICAwAAAAAAAAAAAAABESEAMVFxIEGh/9oACAEBAAE/EC2C3jDJLqP3vJiAcBMnQEPsb3yqCMeJhpBYBtA3hCmmUm42rtdr4f/Z",
            width: 50,
            height: 50,
            src:
              "/static/244120f0b1abdd14a81264c9b9be7356/9b664/logo-kickz.jpg",
            srcSet:
              "/static/244120f0b1abdd14a81264c9b9be7356/9b664/logo-kickz.jpg 1x,\n/static/244120f0b1abdd14a81264c9b9be7356/06a10/logo-kickz.jpg 1.5x,\n/static/244120f0b1abdd14a81264c9b9be7356/f1b5a/logo-kickz.jpg 2x",
          },
        },
      },
      logoBodychange: {
        childImageSharp: {
          fixed: {
            base64:
              "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAFwABAQEBAAAAAAAAAAAAAAAAAAMEAf/EABYBAQEBAAAAAAAAAAAAAAAAAAABAv/aAAwDAQACEAMQAAAByXjq3vELaCOA/8QAGhAAAgMBAQAAAAAAAAAAAAAAAAECAxEQEv/aAAgBAQABBQJLSdeciyy30N6zRn//xAAUEQEAAAAAAAAAAAAAAAAAAAAg/9oACAEDAQE/AR//xAAUEQEAAAAAAAAAAAAAAAAAAAAg/9oACAECAQE/AR//xAAbEAABBQEBAAAAAAAAAAAAAAABABARITFBUf/aAAgBAQAGPwJc8otBxWOzstgWBv/EAB4QAQADAAEFAQAAAAAAAAAAAAEAESFxEDFBUWHw/9oACAEBAAE/IeMHmKOxbppb6fvQeVfDCDs4H4yIitrLKlST3AJ//9oADAMBAAIAAwAAABB82IP/xAAWEQEBAQAAAAAAAAAAAAAAAAARASD/2gAIAQMBAT8Qq4//xAAWEQADAAAAAAAAAAAAAAAAAAABESD/2gAIAQIBAT8QLj//xAAfEAEAAQQCAwEAAAAAAAAAAAABEQAhMWFBUYGRsdH/2gAIAQEAAT8QWbJZK7cG1oIJbsAPR9YpEYSE7oATNome7Za5RIYTDCpwwRfjnVIGVKvLSc78HDrdQufw/tBCFjAV/9k=",
            width: 50,
            height: 50,
            src:
              "/static/324c8674b839289a46e51f9c8cead15a/9b664/logo-bodychange.jpg",
            srcSet:
              "/static/324c8674b839289a46e51f9c8cead15a/9b664/logo-bodychange.jpg 1x,\n/static/324c8674b839289a46e51f9c8cead15a/06a10/logo-bodychange.jpg 1.5x,\n/static/324c8674b839289a46e51f9c8cead15a/f1b5a/logo-bodychange.jpg 2x",
          },
        },
      },
      logoPeru: {
        childImageSharp: {
          fixed: {
            base64:
              "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAGQABAAMBAQAAAAAAAAAAAAAAAAIDBQEE/8QAFgEBAQEAAAAAAAAAAAAAAAAAAAID/9oADAMBAAIQAxAAAAHZqh4402hefIhYD//EABwQAQACAgMBAAAAAAAAAAAAAAEAAgMRBBASIv/aAAgBAQABBQK+QpDLonKDz8rE3CtTr//EABcRAAMBAAAAAAAAAAAAAAAAAAECESD/2gAIAQMBAT8BCWHH/8QAFxEAAwEAAAAAAAAAAAAAAAAAAQIRIP/aAAgBAgEBPwEtDMf/xAAbEAEAAgMBAQAAAAAAAAAAAAABABECECExQf/aAAgBAQAGPwLstbNCtJDByKfuuzzX/8QAHBABAQEAAwADAAAAAAAAAAAAAREAITFBEFFh/9oACAEBAAE/IZ52TeBPmHWGg/eXSji+61YOfWCAYBAJ+5Sgtvx//9oADAMBAAIAAwAAABDQD0D/xAAbEQACAQUAAAAAAAAAAAAAAAABECERMUGB4f/aAAgBAwEBPxCOWzriqRC//8QAFxEBAAMAAAAAAAAAAAAAAAAAASAxUf/aAAgBAgEBPxA9DUP/xAAgEAACAgEDBQAAAAAAAAAAAAABEQAhMUFRgRBhodHw/9oACAEBAAE/EDDDRFVWvkQOrSSWd8Z+MENwANwdxU6mIfviHAUzWROWrNYggmAFOxCgYhQyE5DRO23T/9k=",
            width: 50,
            height: 50,
            src: "/static/d211f0cd11551f065f3276eb69cd7997/9b664/logo-peru.jpg",
            srcSet:
              "/static/d211f0cd11551f065f3276eb69cd7997/9b664/logo-peru.jpg 1x,\n/static/d211f0cd11551f065f3276eb69cd7997/06a10/logo-peru.jpg 1.5x,\n/static/d211f0cd11551f065f3276eb69cd7997/f1b5a/logo-peru.jpg 2x",
          },
        },
      },
    }

    const renderer = new ShallowRenderer()
    const result = renderer.render(<PureMyWorkPage data={data} />)
    expect(result).toMatchSnapshot()
  })
})
