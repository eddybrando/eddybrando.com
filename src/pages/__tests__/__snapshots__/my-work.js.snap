// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyWorkPage renders correctly 1`] = `
<Layout
  headerTitle={
    Object {
      "highlight": "work",
      "prepend": "My ",
    }
  }
  mainContainerClass="main-container--full-height"
  nav={
    Array [
      Object {
        "label": "Home",
        "link": "/",
      },
      Object {
        "label": "Contact me",
        "link": "/contact-me",
      },
    ]
  }
>
  <SEO
    description="This is my work as a developer, designer and entrepreneur specialised in web technologies."
    lang="en"
    meta={Array []}
    noTitleTemplate={false}
    title="My work"
  />
  <section
    className="how-it-began"
  >
    <h2>
      How it began
    </h2>
    <p>
      I started coding around 2003 at age 13 and drafted my first web designs when I was 14.
    </p>
    <p>
      Back then, my tech stack consisted of HTML 4, CSS 2 and Macromedia Flash 7, along with the occasional JavaScript snippet.
    </p>
    <p>
      My tools of choice were Macromedia Dreamweaver and Microsoft FrontPage for code editing, and Yahoo! GeoCities or the free .tk domains host for hosting.
    </p>
  </section>
  <section
    className="work-companies"
  >
    <h2>
      Some companies I've worked at
    </h2>
  </section>
  <div
    className="work-company-block"
  >
    <div>
      <Image
        Tag="div"
        alt=""
        className="company-logo"
        durationFadeIn={500}
        fadeIn={true}
        fixed={
          Object {
            "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAGQABAAIDAAAAAAAAAAAAAAAAAAEDAgQF/8QAFAEBAAAAAAAAAAAAAAAAAAAAAP/aAAwDAQACEAMQAAAB4K+80EjEAH//xAAcEAACAQUBAAAAAAAAAAAAAAABAhAAAxESEzH/2gAIAQEAAQUCAJrVotuFHdcn2f/EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQMBAT8BH//EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQIBAT8BH//EABsQAAEEAwAAAAAAAAAAAAAAAAEAESExECBB/9oACAEBAAY/AoBVHEvbwu6//8QAHRAAAgICAwEAAAAAAAAAAAAAAREAMSFBIHGBkf/aAAgBAQABPyG2HQn0FUyMGKxV+EQwBZF6hMyHe+H/2gAMAwEAAgADAAAAEMAAPP/EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQMBAT8QH//EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQIBAT8QH//EAB0QAQACAgIDAAAAAAAAAAAAAAEAESExQXEQUYH/2gAIAQEAAT8QDUEatHMQaTeRb9dxsbBGkeIoHcIFV9c3MTRagVQqzO9wkiFI7Pcd+Cf/2Q==",
            "height": 50,
            "src": "/static/b6b2d60fa4a246475c6e494ea0cb0867/9b664/logo-ibm.jpg",
            "srcSet": "/static/b6b2d60fa4a246475c6e494ea0cb0867/9b664/logo-ibm.jpg 1x,
/static/b6b2d60fa4a246475c6e494ea0cb0867/06a10/logo-ibm.jpg 1.5x,
/static/b6b2d60fa4a246475c6e494ea0cb0867/f1b5a/logo-ibm.jpg 2x",
            "width": 50,
          }
        }
        loading="lazy"
      />
    </div>
    <div>
      <h3>
        IBM
         
        <span
          className="inline-date"
        >
          (
          2019 - Present
          )
        </span>
      </h3>
      <div
        className="font-small"
      >
        Web developer, UI/UX designer
      </div>
    </div>
    <div
      className="col-date"
    >
      2019 - Present
    </div>
  </div>
  <div
    className="work-company-block"
  >
    <div>
      <Image
        Tag="div"
        alt=""
        className="company-logo"
        durationFadeIn={500}
        fadeIn={true}
        fixed={
          Object {
            "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAGAABAQADAAAAAAAAAAAAAAAAAAMCBAX/xAAUAQEAAAAAAAAAAAAAAAAAAAAA/9oADAMBAAIQAxAAAAHkVtU0WYkAD//EABsQAAIDAAMAAAAAAAAAAAAAAAECAAMSEBET/9oACAEBAAEFAphoQVNfWfStDY235//EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQMBAT8BH//EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQIBAT8BH//EAB0QAAICAQUAAAAAAAAAAAAAAAABAhExICEyYZH/2gAIAQEABj8CMejTyjnTNrk+nSHKq0f/xAAdEAEAAgICAwAAAAAAAAAAAAABACERQSAxUYGh/9oACAEBAAE/IYPdTzSGXhMJFXhuGvsQUUV7A9TbDo4f/9oADAMBAAIAAwAAABCzDzz/xAAUEQEAAAAAAAAAAAAAAAAAAAAg/9oACAEDAQE/EB//xAAUEQEAAAAAAAAAAAAAAAAAAAAg/9oACAECAQE/EB//xAAdEAEAAgICAwAAAAAAAAAAAAABESEAMVFxIEGh/9oACAEBAAE/EC2C3jDJLqP3vJiAcBMnQEPsb3yqCMeJhpBYBtA3hCmmUm42rtdr4f/Z",
            "height": 50,
            "src": "/static/244120f0b1abdd14a81264c9b9be7356/9b664/logo-kickz.jpg",
            "srcSet": "/static/244120f0b1abdd14a81264c9b9be7356/9b664/logo-kickz.jpg 1x,
/static/244120f0b1abdd14a81264c9b9be7356/06a10/logo-kickz.jpg 1.5x,
/static/244120f0b1abdd14a81264c9b9be7356/f1b5a/logo-kickz.jpg 2x",
            "width": 50,
          }
        }
        loading="lazy"
      />
    </div>
    <div>
      <h3>
        KICKZ
         
        <span
          className="inline-date"
        >
          (
          2018 - 2019
          )
        </span>
      </h3>
      <div
        className="font-small"
      >
        Web developer, architect, UI/UX designer
      </div>
    </div>
    <div
      className="col-date"
    >
      2018 - 2019
    </div>
  </div>
  <div
    className="work-company-block"
  >
    <div>
      <Image
        Tag="div"
        alt=""
        className="company-logo"
        durationFadeIn={500}
        fadeIn={true}
        fixed={
          Object {
            "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAGQABAAIDAAAAAAAAAAAAAAAAAAEDAgQF/8QAFAEBAAAAAAAAAAAAAAAAAAAAAP/aAAwDAQACEAMQAAAB4K+80EjEAH//xAAcEAACAQUBAAAAAAAAAAAAAAABAhAAAxESEzH/2gAIAQEAAQUCAJrVotuFHdcn2f/EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQMBAT8BH//EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQIBAT8BH//EABsQAAEEAwAAAAAAAAAAAAAAAAEAESExECBB/9oACAEBAAY/AoBVHEvbwu6//8QAHRAAAgICAwEAAAAAAAAAAAAAAREAMSFBIHGBkf/aAAgBAQABPyG2HQn0FUyMGKxV+EQwBZF6hMyHe+H/2gAMAwEAAgADAAAAEMAAPP/EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQMBAT8QH//EABQRAQAAAAAAAAAAAAAAAAAAACD/2gAIAQIBAT8QH//EAB0QAQACAgIDAAAAAAAAAAAAAAEAESExQXEQUYH/2gAIAQEAAT8QDUEatHMQaTeRb9dxsbBGkeIoHcIFV9c3MTRagVQqzO9wkiFI7Pcd+Cf/2Q==",
            "height": 50,
            "src": "/static/b6b2d60fa4a246475c6e494ea0cb0867/9b664/logo-ibm.jpg",
            "srcSet": "/static/b6b2d60fa4a246475c6e494ea0cb0867/9b664/logo-ibm.jpg 1x,
/static/b6b2d60fa4a246475c6e494ea0cb0867/06a10/logo-ibm.jpg 1.5x,
/static/b6b2d60fa4a246475c6e494ea0cb0867/f1b5a/logo-ibm.jpg 2x",
            "width": 50,
          }
        }
        loading="lazy"
      />
    </div>
    <div>
      <h3>
        IBM
         
        <span
          className="inline-date"
        >
          (
          2016 - 2018
          )
        </span>
      </h3>
      <div
        className="font-small"
      >
        Web developer
      </div>
    </div>
    <div
      className="col-date"
    >
      2016 - 2018
    </div>
  </div>
  <div
    className="work-company-block"
  >
    <div>
      <Image
        Tag="div"
        alt=""
        className="company-logo"
        durationFadeIn={500}
        fadeIn={true}
        fixed={
          Object {
            "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAFwABAQEBAAAAAAAAAAAAAAAAAAMEAf/EABYBAQEBAAAAAAAAAAAAAAAAAAABAv/aAAwDAQACEAMQAAAByXjq3vELaCOA/8QAGhAAAgMBAQAAAAAAAAAAAAAAAAECAxEQEv/aAAgBAQABBQJLSdeciyy30N6zRn//xAAUEQEAAAAAAAAAAAAAAAAAAAAg/9oACAEDAQE/AR//xAAUEQEAAAAAAAAAAAAAAAAAAAAg/9oACAECAQE/AR//xAAbEAABBQEBAAAAAAAAAAAAAAABABARITFBUf/aAAgBAQAGPwJc8otBxWOzstgWBv/EAB4QAQADAAEFAQAAAAAAAAAAAAEAESFxEDFBUWHw/9oACAEBAAE/IeMHmKOxbppb6fvQeVfDCDs4H4yIitrLKlST3AJ//9oADAMBAAIAAwAAABB82IP/xAAWEQEBAQAAAAAAAAAAAAAAAAARASD/2gAIAQMBAT8Qq4//xAAWEQADAAAAAAAAAAAAAAAAAAABESD/2gAIAQIBAT8QLj//xAAfEAEAAQQCAwEAAAAAAAAAAAABEQAhMWFBUYGRsdH/2gAIAQEAAT8QWbJZK7cG1oIJbsAPR9YpEYSE7oATNome7Za5RIYTDCpwwRfjnVIGVKvLSc78HDrdQufw/tBCFjAV/9k=",
            "height": 50,
            "src": "/static/324c8674b839289a46e51f9c8cead15a/9b664/logo-bodychange.jpg",
            "srcSet": "/static/324c8674b839289a46e51f9c8cead15a/9b664/logo-bodychange.jpg 1x,
/static/324c8674b839289a46e51f9c8cead15a/06a10/logo-bodychange.jpg 1.5x,
/static/324c8674b839289a46e51f9c8cead15a/f1b5a/logo-bodychange.jpg 2x",
            "width": 50,
          }
        }
        loading="lazy"
      />
    </div>
    <div>
      <h3>
        BodyChange
         
        <span
          className="inline-date"
        >
          (
          2014 - 2015
          )
        </span>
      </h3>
      <div
        className="font-small"
      >
        Web developer, UI designer, project manager
      </div>
    </div>
    <div
      className="col-date"
    >
      2014 - 2015
    </div>
  </div>
  <div
    className="work-company-block"
  >
    <div>
      <Image
        Tag="div"
        alt=""
        className="company-logo"
        durationFadeIn={500}
        fadeIn={true}
        fixed={
          Object {
            "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAUABQDASIAAhEBAxEB/8QAGQABAAMBAQAAAAAAAAAAAAAAAAIDBQEE/8QAFgEBAQEAAAAAAAAAAAAAAAAAAAID/9oADAMBAAIQAxAAAAHZqh4402hefIhYD//EABwQAQACAgMBAAAAAAAAAAAAAAEAAgMRBBASIv/aAAgBAQABBQK+QpDLonKDz8rE3CtTr//EABcRAAMBAAAAAAAAAAAAAAAAAAECESD/2gAIAQMBAT8BCWHH/8QAFxEAAwEAAAAAAAAAAAAAAAAAAQIRIP/aAAgBAgEBPwEtDMf/xAAbEAEAAgMBAQAAAAAAAAAAAAABABECECExQf/aAAgBAQAGPwLstbNCtJDByKfuuzzX/8QAHBABAQEAAwADAAAAAAAAAAAAAREAITFBEFFh/9oACAEBAAE/IZ52TeBPmHWGg/eXSji+61YOfWCAYBAJ+5Sgtvx//9oADAMBAAIAAwAAABDQD0D/xAAbEQACAQUAAAAAAAAAAAAAAAABECERMUGB4f/aAAgBAwEBPxCOWzriqRC//8QAFxEBAAMAAAAAAAAAAAAAAAAAASAxUf/aAAgBAgEBPxA9DUP/xAAgEAACAgEDBQAAAAAAAAAAAAABEQAhMUFRgRBhodHw/9oACAEBAAE/EDDDRFVWvkQOrSSWd8Z+MENwANwdxU6mIfviHAUzWROWrNYggmAFOxCgYhQyE5DRO23T/9k=",
            "height": 50,
            "src": "/static/d211f0cd11551f065f3276eb69cd7997/9b664/logo-peru.jpg",
            "srcSet": "/static/d211f0cd11551f065f3276eb69cd7997/9b664/logo-peru.jpg 1x,
/static/d211f0cd11551f065f3276eb69cd7997/06a10/logo-peru.jpg 1.5x,
/static/d211f0cd11551f065f3276eb69cd7997/f1b5a/logo-peru.jpg 2x",
            "width": 50,
          }
        }
        loading="lazy"
      />
    </div>
    <div>
      <h3>
        Peruvian Embassy in Ecuador
         
        <span
          className="inline-date"
        >
          (
          2008 - 2015
          )
        </span>
      </h3>
      <div
        className="font-small"
      >
        Web developer, UI/UX designer, graphic designer
      </div>
    </div>
    <div
      className="col-date"
    >
      2008 - 2015
    </div>
  </div>
</Layout>
`;
